pkgs/servers/mail/dovecot/default.nix

```diff
@@ -62,6 +62,7 @@ stdenv.mkDerivation rec {
     "--with-lucene"
     "--with-icu"
   ] ++ lib.optional (stdenv.isLinux) "--with-systemdsystemunitdir=$(out)/etc/systemd/system"
+    ++ lib.optional (stdenv.isDarwin) "--enable-static"
```

Comment:
Currently building dovecot from vanilla without '--enable-static' breaks in libtool calls. I've tested the build outside of stdenv.darwin on an osx with xcode tools to validate this, too. Until a patch is available for dovecot, it is the only solution to get dovecot on osx afaik.